### PR TITLE
Linux: debug.r_brk function might start with endbr instruction.

### PR DESCRIPTION
**Pull request by d0now**
https://github.com/googleprojectzero/TinyInst/issues/70

### CODE REVIEW

**Comment by d0now** [comment]
need to be tested on intel cet disabled loader.

File: Linux/debugger.cpp [hunk 1]
```diff
@@ -1032,7 +1032,15 @@ int Debugger::GetLoadedModulesT(std::set<LoadedModule> &modules, bool set_breakp
   }
 
   if(set_breakpoint) {
-    AddBreakpoint((void *)(uint64_t)debug.r_brk, BREAKPOINT_NOTIFICATION); 
+    uint32_t inst = 0;
```
Comment (ifratric) [inline Linux/debugger.cpp:1035]: This should work, but note that the extra steps should only be performed on x86/x64 CPUs. On ARM64 (`#ifdef ARM64`) the old behavior should be kept.

Comment (d0now) [reply Linux/debugger.cpp:1038]: I've just add new commit. Can you check it please? :)

**Comment by ifratric** [comment]
Great, thanks!

**Pull request merged**